packages/blade/src/components/TopNav/TopNav.web.tsx

```diff
@@ -73,21 +40,19 @@ const TopNav = ({ children, ...styledProps }: TopNavProps): React.ReactElement =
     <TopNavContext.Provider value={{ backgroundColor: styledProps.backgroundColor }}>
       <Box
         display="grid"
-        gridTemplateColumns={{ base: 'minmax(0, 1fr) auto', m: 'auto minmax(0, 1fr) auto' }}
+        gridTemplateColumns="auto minmax(0, 1fr) auto"
         alignItems="center"
         position="sticky"
         top="0px"
         width="100%"
-        paddingX="spacing.3"
+        paddingY={{ base: 'spacing.3', m: 'spacing.0' }}
+        paddingX={{ base: 'spacing.4', m: 'spacing.3' }}
         height={makeSize(TOP_NAV_HEIGHT)}
         zIndex={1}
         {...styledProps}
```

Comment:
```suggestion
        {...getStyledProps(styledProps)}
```

Reply:
We actually also want to support other box props like backgroundColor, width etc so give flexibility to users since it's just a Box like container thus not doing `getStyledProps`

Reply:
Oh I guess call it `boxProps` then

Reply:
done